src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import { gql } from 'apollo-boost';
+import { useQuery } from '@apollo/react-hooks';
+import classNames from 'classnames';
+
+import { getCommaSeparatedNumber } from 'src/shared/helpers/formatters/numberFormatter';
+import {
+  getFeatureCoordinates,
+  getRegionName
+} from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
+
+import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
+
+import transcriptsListStyles from '../DefaultTranscriptsList.scss';
+import styles from './TranscriptsListItemInfo.scss';
+
+type ItemInfoProps = {
+  transcriptId: string | null;
+};
+
+type ItemInfoWithDataProps = {
+  transcript: Transcript;
+};
+
+const QUERY = gql`
+  query Transcript($id: String!) {
+    transcript(byId: { id: $id }) {
+      id
+      symbol
+      biotype
+      slice {
+        location {
+          start
+          end
+        }
+        region {
+          name
+          strand {
+            code
+          }
+        }
+      }
+      exons {
+        slice {
+          location {
+            start
+            end
+          }
+        }
+      }
+      cds {
+        start
+        end
+      }
+    }
+  }
+`;
```

Comment:
Most of this has likely already been queried and is accessible from GeneView component.

Reply:
The transcript data now comes from the `GeneView` component. So the loading problem should be resolved too.

Reply:
✔️